ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh

```diff
@@ -943,14 +943,14 @@ function run_upgrade_e2e() {
     parallel_cucumber -n "${PARALLEL}" --first-is-1 --type cucumber --serialize-stdout --combine-stderr --prefix-output-with-test-env-number --exec \
     'export OPENSHIFT_ENV_OCP4_USER_MANAGER_USERS=$(echo ${USERS} | cut -d "," -f ${TEST_ENV_NUMBER},$((${TEST_ENV_NUMBER}+${PARALLEL})),$((${TEST_ENV_NUMBER}+${PARALLEL}*2)));
      export WORKSPACE=/tmp/dir${TEST_ENV_NUMBER};
-     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and not @admin\" -p junit"' || true
+     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and not @admin\" -p junit"' &>> "${ARTIFACT_DIR}/e2etest-log.txt" || true
```

Comment:
```suggestion
     parallel_cucumber --group-by found --only-group ${TEST_ENV_NUMBER} -o "--tags \"${E2E_RUN_TAGS} and not @admin\" -p junit"' &>> "${ARTIFACT_DIR}/e2etest-log-${e2e_start_time}.txt" || true
```

Let us append timestamp for the log file to avoid the file size get too big.

Reply:
good idea, but due to there are both normal log and admin log with the same timestamp ${e2e_start_time}, so I plan to separate it into `e2e-normal-log.txt` and `e2e-admin-log.txt`, wdyt?

Reply:
I am okay with merging both normal log and admin log into one file, we just want to avoid the file size getting too big.

Reply:
If those e2e tests are executed multiple times, e.g, for 4.12 upgrade to 4.16, for 4 or 5 times, perhaps we can also append the current OCP version to the log name ? So we have `logfilename-4.12.txt, logfilename-4.13.txt, ..., logfilename-4.16.txt`

Reply:
hmm, do u mean keep all e2e logs in one file with ${e2e_start_time} postfix, then how to reduce the file size?

Reply:
`run_upgrade_e2e` is being called by several times, in each call, `e2e_start_time` will get changed, so the log files will be created for several times with different timestamp.

Reply:
got both of you now, let me update to use timestamp and add ver for better reading then.